department-of-veterans-affairs/caseflow
# Update virtual hearings email templates with internal use section + other changes

Resolves https://vajira.max.gov/browse/CASEFLOW-508

### Description

  - Update subject line with do not reply
  - Make modifications to "Test Your Connection" section
  - Add internal use section at the bottom of every email
  - Do not to UtilityFunction check in helpers folder
  - Add / update rspec tests

### Acceptance Criteria
- [x] Add a "For internal Board use:" section to the bottom of the confirmation and reminder emails sent to the Veteran and the representative that includes
  - [x] a two-paragraph height gap between "The Board of Veterans' Appeals" and the text
  - [x] the two-letter abbreviation of the state where the appellant resides
- [x] a link to the Case Details page for the appeal in Caseflow with the 
example: For internal Board use: IL CF
- [x] Add " – Do Not Reply" to the end of the subject
- [x] Add the following text to the bottom of the "Test Your Connection" section `Want to learn more about Virtual Tele-hearings? Watch this video at https://www.youtube.com/watch?v=UDI3C_ytJt8.`

### Testing Plan

- Run `bundle exec rake emails:virtual_hearings:sample` and spot check

### User Facing Changes

![_Users_ferristseng_Downloads_Sample%20Emails_confirmation_Appellant html](https://user-images.githubusercontent.com/1298274/105224958-e34b0d00-5b2b-11eb-8e9c-244885c914f9.png)
